### PR TITLE
Update action to exit gracefully

### PR DESCRIPTION
**Pull request by Seia-Soto**
This PR fixes issue #5 and make the workflow script work on any repository by using `github` variable.

- Actual test: https://github.com/Seia-Soto/awscliv2.appimage/actions/runs/450762282
- Test that validates the correct work result of the job level `if statement`: https://github.com/Seia-Soto/awscliv2.appimage/actions/runs/450753899

### CODE REVIEW

File: scripts/autobuild/install-deps.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+sudo apt-get update
+sudo apt-get install git unzip curl jq libfuse-dev -y
```
Comment (simnalamburt) [inline scripts/autobuild/install-deps.sh:2]: libfuse-dev 만 남기고 지워도 될거같아요

##### Reference
- https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#installed-apt-packages

File: .github/workflows/autobuild.yml [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+  check-version:
     runs-on: ubuntu-latest
+    outputs:
+      remote: ${{ steps.latest_remote_tag.outputs.ver }}
+      local: ${{ steps.latest_local_release.outputs.ver }}
     steps:
       - uses: actions/checkout@v2
       - name: Install dependencies
-        run: sudo apt-get update && sudo apt-get install git unzip curl jq libfuse-dev -y
+        run: bash ./scripts/autobuild/install-deps.sh
```
Comment (simnalamburt) [inline .github/workflows/autobuild.yml:16]: 이 라인은 필요 없을거같아요

File: .github/workflows/autobuild.yml [hunk 3]
```diff
@@ -23,38 +26,26 @@ jobs:
...
+  build:
+    runs-on: ubuntu-latest
+    needs: [check-version]
+    if: |
+      (needs.check-version.outputs.remote != needs.check-version.outputs.local) &&
+      startsWith(needs.check-version.outputs.remote, '2.')
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: bash ./scripts/autobuild/install-deps.sh
```
Comment (simnalamburt) [inline .github/workflows/autobuild.yml:43]: 별도 파일로 빼지 않고 인라인해도 될거같아요

**Review by simnalamburt — CHANGES_REQUESTED** [review]

**Comment by Seia-Soto** [comment]
수정 완료했습니다!

Changes were displayed.

**Review by simnalamburt — APPROVED** [review]

**Pull request merged**